pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```
@@ -40,6 +40,15 @@ let
         let
           grammars = map grammarToPlugin
             (f (tree-sitter.builtGrammars // builtGrammars));
+          filterDuplicateGrammars = lib.lists.foldl' (
+            res: new-el:
+              if lib.lists.any (old-el:
+                # `meta.name` contains something like `c-grammar-0.0.0+rev=be23d2c`
+                new-el.meta.name == old-el.meta.name
```

```suggestion
                lib.getName new-el == lib.getName old-el
```

This seems to solve the error with norg for me

---

so yeah if you dont wanna `ln -sf`, which, fair enough I didnt really either, this seems to be the thing that works. Tried a couple different conditions but this one is the way used all over neovimUtils and it works, so, seems good to me.